extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -115,6 +116,30 @@ export class Bind {
   }
 
   /**
+   * Same as `setState()` except this method supports keys nested with '.'.
```

Comment:
Does it make sense to have two APIs? Is it not possible to contain everything in one?